modules/experiment/src/test/java/com/intuit/wasabi/experiment/impl/ExperimentsImplTest.java

```diff
@@ -133,29 +134,44 @@ public void testCreateExperimentFailedValidatorPersonalization() {
                 .withEndTime(endTime).build();
         testExp.setApplicationName(Application.Name.valueOf(""));
         assertThat(testExp.getApplicationName(), is(Application.Name.valueOf("")));
+
+        //Actual call
         expImpl.createExperiment(testExp, UserInfo.from(UserInfo.Username.valueOf("user")).build());
+
```

Comment:
@abhishekjain88 Looked into details and found that IllegalArgumentsException is coming from NewExperiment.build() method itsetlf and this was not really unit test for ExperimentImpl.createExperiment() so removed this test and added new one NewExperimentTest.testExperimentPartialPersonalizationInfo()